app/Actions/DisplayHelpScreen.php

```diff
@@ -18,20 +16,23 @@ class DisplayHelpScreen
 
     public function __invoke()
     {
+        $consoleWriter = app('console-writer');
 
-        $this->line("\n<comment>Usage:</comment>");
-        $this->line("  lambo new myApplication [arguments]\n");
-        $this->line("<comment>Commands (lambo COMMANDNAME):</comment>");
+        $consoleWriter->newLine();
+        $consoleWriter->ignoreVerbosity()->text("<comment>Usage:</comment>");
```

Comment:
What if we re-work `ignoreVerbosity` to instead be a `verbose()` modifier on anything that should only output on verbose mode?

Reply:
Done